CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # CHANGELOG
 
+## 2.6.0 (2023-08-15)
+
+MLflow 2.6.0 includes several major features and improvements
+
+Features:
+
+- [Tracking] Support `save_kwargs` for `mlflow.log_figure` to specify extra options when saving a figure (#9179, @stroblme)
+- [Gateway] Add support for MLflow serving to MLflow AI Gateway (#9199, @BenWilson2)
+- [Artifacts] Display progress bars when uploading/download artifacts (#9195, @serena-ruan)
+- [Models] Support logging retriever models in langchain (#8808, @liangz1)
+- [Tracking] Add support to log customized tags to runs created by autologging (#9114, @thinkall)
+- [Models / Scoring] Add support for passing extra params during inference for PyFunc models (#9068, @serena-ruan)
+
+Bug fixes:
+
+- [Models] Fix text_pair functionality for transformers TextClassification pipelines (#9215, @BenWilson2)
+- [Models] Fix LangChain compatibility with SQLDatabase (#9192, @dbczumar)
+- [Tracking] Fix sklearn flavor `_get_metric_name_list` function (#9095, @WeichenXu123)
```

Comment:
```suggestion
- [Tracking] Avoid patching `sklearn.metrics.get_scorer_names` in `mlflow.sklearn.autolog` to avoid duplicate logging (#9095, @WeichenXu123)
```